pages/Documents/RichMessaging/AppleBusinessChatTemplates/custom-interactive-message-template.md

```diff
@@ -0,0 +1,248 @@
+
+Please note that the structured content body layout will **not** be represented on the consumer device. The body will help you to create a visual representation in Conversational Cloud for conversational context, transcript and historic records, as well as ease of use for agents.
+
+In order to allow you to report on the number of times an iMessage app was sent from an agent widget during a conversation, make sure to add a unique identifier in the structured content body for each type of iMessage app you use. This will allow you to search transcripts and track how many times a specific app was sent.
+
+For the structured content body fields descriptions and different layout options please refer to [Getting Started with Structured Content.](getting-started-with-rich-messaging-introduction.html)
+
+### Metadata Template
+
+The structured content metadata will allow you to define which iMessage app / extension should be sent to the consumer and which custom data to pass into the iMessage app / extension when it loads.
+
+Using Metadata, you can also control the visual appearance of the bubble that displays to the consumer. This is done with the `receivedMessage` object within the metadata.
+
+#### Metadata Properties
+
+<table>
+  <thead>
+    <td>Property Name</td>
+    <td>Description</td>
+    <td>Type</td>
+    <td>Required</td>
+  </thead>
+  <tbody>
+  <tr>
+    <td>type</td>
+    <td>Type of metadata.                              Must be BusinessChatCustomMessage </td>
+    <td>Enum</td>
+    <td>Y</td>
+  </tr>
+  <tr>
+    <td>appId</td>
+    <td>The App Store identifier of the custom                                 Messages framework extension </td>
+    <td>Integer</td>
+    <td>N</td>
+  </tr>
+  <tr>
+    <td>appName</td>
+    <td>The name of the custom Messages                               framework extension.</td>
+    <td>String</td>
+    <td>N</td>
+  </tr>
+  <tr>
+    <td>bid</td>
+    <td>Identifies the Messages framework extension that the customer interacts with while using Messages. It is composed along with the extension bundleID, and teamID </td>
+    <td>String</td>
+    <td>Y</td>
+  </tr>
+  <tr>
+    <td>sessionIdentifier</td>
+    <td>Valid UUID string, you must generate. This could represent a message or a group or messages that identifies the session or transaction</td>
+    <td>String</td>
+    <td>N</td>
+  </tr>
+  <tr>
+    <td>useLiveLayout</td>
+    <td>A flag that determines if Live Layout is enabled. The default is false.</td>
+    <td>Boolean</td>
+    <td>N</td>
+  </tr>
+  <tr>
+    <td>receivedMessage</td>
+    <td>Bubble displayed when interactive message is received and LiveLayout is set to false</td>
+    <td>Object</td>
+    <td>Y</td>
+  </tr>
+  <tr>
+    <td>URL</td>
+    <td>The URL to the data that is sent to the Messages framework extension. Should be sent encoded</td>
+    <td>String</td>
+    <td>Y</td>
+  </tr>
+  </tbody>
+</table>
+
+
+#### Received Message Properties
+
+<table>
```

Comment:
Please use Markdown.

Reply:
I only edited this page -- can you help me understand how to translate it into markdown?